source/whats-new.txt

```diff
@@ -1 +1,18 @@
-TODO: write
+.. _mongodb-analyzer-changelog:
+
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+What's New in 1.0
```

Comment:
Could be helpful to have a label here for direct linking for when you have multiple versions.

Reply:
Added a ref!